components/Activities/Activities.tsx

```diff
@@ -32,7 +32,7 @@ const Activities: React.FC = () => {
                     alt={activity.alt}
                     src={require(`@/content/activities/${activity.imagePath}?width=600&height=400`)}
                     data-loaded="true"
-                    width={350}
+                    width={540}
                     height={233.33}
```

Comment:
```suggestion
                    width={540}
                    height={360}
```

幅:高さが3:2だったみたいなので、540に合わせて修正し欲しいです 🙇